src/main/java/dk/kb/present/DSCollection.java

```diff
@@ -175,9 +175,9 @@ public Stream<DsRecordDto> getDSRecords(
         log.debug("Calling storage.getDSRecords(origin='{}', mTime={}, maxRecords={})",
                 origin, mTime, maxRecords);
         try {
-            // 500 is a magic number, which is poor code style. Currently, it controls batch size against ds-license
+            // 35 is a magic number, which is poor code style. Currently, it controls batch size against ds-license
             return ExtractionUtils.splitToLists(
-                            storage.getDSRecordsByRecordTypeLocalTree(origin, recordRequestType, mTime, maxRecords), 500)
+                            storage.getDSRecordsByRecordTypeLocalTree(origin, recordRequestType, mTime, maxRecords), 35)
```

Comment:
Note to self @VictorHarbo : Increase this when the problem has been fixed in the `ds-license` end.